experimental/dds/tree/api-report/experimental-tree.alpha.api.md

```diff
@@ -361,9 +361,6 @@ export interface ForestNode extends NodeData<NodeId> {
     readonly traits: ReadonlyMap<TraitLabel, readonly NodeId[]>;
 }
 
-export { getSerializedUploadedEditChunkContents }
-export { getSerializedUploadedEditChunkContents as getUploadedEditChunkContents }
```

Comment:
`getSerializedUploadedEditChunkContents` has been `@internal` ever since we created the tag, so seems they were incorrect in showing up here.